lms/djangoapps/instructor_task/config/models.py

```diff
@@ -0,0 +1,14 @@
+"""
+Models for configuration of settings relevant
+to instructor tasks.
+"""
+from config_models.models import ConfigurationModel
+from django.db.models import IntegerField
+
+
+class GradeReportsBatchSizeSetting(ConfigurationModel):
```

Comment:
Since the sql table name is taken from the model class name, consider using a more generic model class name so it can evolve with time and future changes; but not too generic that the table becomes unusable/undiscoverable.  For example, scoping this table to `GradeReportSetting` could do the trick.  What do you think?